rules/experimental/rbe/exec_properties.bzl

```diff
@@ -13,6 +13,92 @@
 # limitations under the License.
 
 """This file contains macros to create and manipulate dictionaries of properties to be used as execution properties for RBE.
+
+It also contains macros that create repository rules for standard and custom sets of execution
+property sets.
+
+Here are some examples of how to use these repository rules:
+
+Scenario 1 - The standard use case:
+
+In the WORKSPACE file, call
+  rbe_exec_properties(
+      name = "exec_properties",
+  )
+
+This creates a local repo @rbe_exec_properties with standard RBE execution property sets. For
+example, NETWORK_ON which is the dict {"dockerNetwork" : "standard"}
+
+Then, in some BUILD file, you can reference this execution property set as follows:
+
+  load("@exec_properties//:exec_propert_sets.bzl", "NETWORK_ON")
+  ...
+  exec_properties = NETWORK_ON
+
+The reason not to directly set exec_properties = {...} in a target is that then it might be hard to
+depend on such a target from another repo, if, say, that other repo wants to use remote execution
+but not RBE.
+
+Scenario 2 - non-RBE remote execution:
```

Comment:
How would this apply for extending a repo with local execution only, extending from a repo that uses @exec_properties? Is it relevant to include that scenario here?

Reply:
Local execution ignores the exec_properties field completely, so it doesn't actually matter how it is populated.
I will add a scenario that states that explicitly.